### PR TITLE
Update DCP_change_log.md

### PR DESCRIPTION
**Pull request by JordyGit**

### CODE REVIEW

File: release-notes/Cloud/DCP_change_log.md [hunk 1]
```diff
@@ -9,6 +9,29 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 26 November 2024 - New Feature - Admin app - Automation Usage [ID ]
+
+From now on, the usage page in the [Admin app](https://admin.dataminer.services) will provide usage data about the Automation script runs when available.
```
Comment (JordyGit) [inline release-notes/Cloud/DCP_change_log.md:14]:
```suggestion
From now on, the usage page in the [Admin app](https://admin.dataminer.services) will provide usage data about Automation script runs when available.
```

File: release-notes/Cloud/DCP_change_log.md [hunk 2]
```diff
@@ -9,6 +9,29 @@ The dataminer.services platform gets updated continuously. This change log can h
...
+From now on, you can create an API key on organization level with the "Retrieve usage data' permission. This API key can be used with the new Key Usage API, in combination with the new Public Usage API, to retrieve usage data about your DataMiner Systems in an automated way.
+
+Find the swagger documentation pages about the available Usage API calls via the following references:
+- [Public Usage API swagger documentation](https://api.dataminer.services/swagger/usageapi/index.html?urls.primaryName=Public+Usage+Api+v1.0)
+  - Get the features were usage data might be available for
+  - Get the metrics of a feature
+- [Key Usage API swagger documentation](https://api.dataminer.services/swagger/usageapi/index.html?urls.primaryName=Key+Usage+Api+v1.0)
+  - Get the data in a given time range for a given feature, metric and granularity with the option to filter & split data on the dimensions or DataMiner Systems
+
+### 26 November 2024 - Enhancement - Admin app - Usage export to CSV email layout [ID ]
```
Comment (JordyGit) [inline release-notes/Cloud/DCP_change_log.md:27]:
```suggestion
### 26 November 2024 - Enhancement - Admin app - Usage export email layout [ID ]
```

File: release-notes/Cloud/DCP_change_log.md [hunk 3]
```diff
@@ -9,6 +9,29 @@ The dataminer.services platform gets updated continuously. This change log can h
...
+
+From now on, the emails with the download link for usage exports will be in the familiar email layout.
+
+### 26 November 2024 - Enhancement - Admin app - Audit export to CSV email layout [ID ]
```
Comment (JordyGit) [inline release-notes/Cloud/DCP_change_log.md:31]:
```suggestion
### 26 November 2024 - Enhancement - Admin app - Audit export email layout [ID ]
```

**Review by AntonPlanckeSLC — APPROVED** [review]

File: release-notes/Cloud/DCP_change_log.md [hunk 4]
```diff
@@ -9,6 +9,26 @@ The dataminer.services platform gets updated continuously. This change log can h
...
+
+From now on, you can create an API key on organization level with the "Retrieve usage data" permission. This API key can be used with the new Key Usage API, in combination with the new Public Usage API, to retrieve usage data about your DataMiner Systems in an automated way.
+
+The swagger documentation pages about the available Usage API calls are available in the following locations:
+
+- [Public Usage API swagger documentation](https://api.dataminer.services/swagger/usageapi/index.html?urls.primaryName=Public+Usage+Api+v1.0)
+  - Get the features for which usage data might be available.
+  - Get the metrics of a feature.
+- [Key Usage API swagger documentation](https://api.dataminer.services/swagger/usageapi/index.html?urls.primaryName=Key+Usage+Api+v1.0)
+  - Get the data in a given time range for a given feature, metric and granularity with the option to filter and split data on the dimensions or DataMiner Systems.
```
Comment (MariekeGO) [inline release-notes/Cloud/DCP_change_log.md:26]: Could you clarify this sentence? It isn't clear to me what you mean with "metric and granularity" and "data on the dimensions".

File: release-notes/Cloud/DCP_change_log.md [hunk 5]
```diff
@@ -9,6 +9,26 @@ The dataminer.services platform gets updated continuously. This change log can h
...
+
+### 26 November 2024 - Enhancement - Admin app - Usage and audit export email layout [ID 41554]
+
+From now on, the emails with the download link for usage exports or audit exports will be in the familiar email layout.
```
Comment (MariekeGO) [inline release-notes/Cloud/DCP_change_log.md:30]: What do you mean with "the familiar email layout"? Could you rephrase this so it's also clear for people who might not know this?

Comment (JordyGit) [reply release-notes/Cloud/DCP_change_log.md:30]: This email existed but as plain text without template/layout like our other emails do (like live sharing emails, expiring daas emails etc)

File: release-notes/Cloud/DCP_change_log.md (hunk 4 above)
Comment (JordyGit) [reply release-notes/Cloud/DCP_change_log.md:26]: these are inputs for the api call 
image

Feature examples: Automation, Storage as a Service,...
Metric examples: Script Runs, Operations,....
Granularity options: Daily, Weekly or Monthly
Dimension splitter examples: split usage data on 'Script Name', split usage data on script 'Succeeded', ...

File: release-notes/Cloud/DCP_change_log.md (hunk 5 above)
Comment (MariekeGO) [reply release-notes/Cloud/DCP_change_log.md:30]:
```suggestion
From now on, the emails with the download link for usage exports or audit exports will use the same template as other emails sent from dataminer.services.
```

**Review by MariekeGO — APPROVED** [review]
